cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlTest.java

```diff
@@ -49,19 +54,21 @@ public void testSanityCheckDryRun() throws InterruptedException, ExecutionExcept
     // For sanityCheckDryRun(false, true) and sanityCheckDryRun(false, false) (see #1 and #2 below).
     EasyMock.expect(executor.hasOngoingExecution()).andReturn(true).times(2);
     // For sanityCheckDryRun(false, XXX) (see #3 below)
+    Set<TopicPartition> dummyOngoingPartitionReassignments = new HashSet<>();
+    dummyOngoingPartitionReassignments.add(DUMMY_TOPIC_PARTITION);
```

Comment:
Nit: Since this is a singleton, can we use
```
Set<TopicPartition> dummyOngoingPartitionReassignments = Collections.singleton(DUMMY_TOPIC_PARTITION);
```